ripple/src/main/test/auth.test.js

```diff
@@ -0,0 +1,21 @@
+/* Contains unit tests for auth.js file. Currently has ~10% coverage (rest will be tested by Sarah).*/
+
+const authModule = require('../webapp/auth');
+const $ = require('jquery');
+require('../__mocks__/localstorage');
+
+/* Test that checks if localStorage correctly stores values if a user registers as business owner. 
+   Local storage only stores strings, so had to add check for boolean and string types */
+test('setUserType: set user type when signing up', () => {
+  authModule.setUserType("business");
+  expect(localStorage.getItem("isBusinessOwner") === "true" || localStorage.getItem("isBusinessOwner") === true).toBeTruthy;
```

Comment:
aren't these both the same? not sure I understand what is being tested here

Reply:
I was running into issues with localStorage not consistently converting the values because it can only store string keys so sometimes the boolean value would be converted to a string. It isn't actually an issue in our implementation so I did this as a workaround. It is confusing though, so I can try finding a better way to write this!

Reply:
Actually, I just tested removing the === true part and only used .toBeTruthy to check and it worked!